testing/trino-product-tests/src/main/resources/tempto-configuration.yaml

```diff
@@ -149,6 +149,7 @@ databases:
   spark:
     host: spark
     jdbc_driver_class: org.apache.hive.jdbc.HiveDriver
+    jdbc_jar: /docker/jdbc/hive-jdbc.jar
```

Comment:
Why do we need this now?

`org.apache.hive.jdbc.HiveDriver` in `hive-jdbc.jar` is same as on the classpath

https://github.com/trinodb/trino/blob/8969ecfa20c39df4cf9ce1ef384c010636365d1e/testing/trino-product-tests-launcher/pom.xml#L188-L200

https://github.com/trinodb/trino/blob/8969ecfa20c39df4cf9ce1ef384c010636365d1e/testing/trino-product-tests-launcher/pom.xml#L302-L309

Reply:
`trino-product-tests` module contains another Hive JDBC driver. New Hive JDBC driver doesn't work with old Hive servers if I remember correctly. I'm losing the context of this PR. This comment might be wrong.
```xml
        <dependency>
            <groupId>io.trino.hive</groupId>
            <artifactId>hive-apache-jdbc</artifactId>
            <scope>runtime</scope>
        </dependency>
```